ajax/libs/d3-shape/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "d3-shape",
+  "filename": "d3-shape.min.js",
+  "description": "Graphical primitives for visualization, such as lines and areas.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "graphics",
+    "visualization",
+    "canvas",
+    "svg"
+  ],
+  "homepage": "https://d3js.org/d3-shape/",
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Mike Bostock",
+    "url": "https://bost.ocks.org/mike"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d3/d3-shape.git"
+  },
+  "npmName": "d3-shape",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
```

Comment:
Please use `**/!(package.js|version.js)` here, package.js appeared in v0.6.1 and version.js appeared in v0.6.0.